clients/csharp-uwp/UWPVoiceAssistantSample/MainPage.xaml.cs

```diff
@@ -213,6 +215,7 @@ private async Task UpdateUIForSharedStateAsync()
             // UI changes must be performed on the UI thread.
             await this.Dispatcher.RunAsync(CoreDispatcherPriority.Normal, async () =>
             {
+                this.ReadBufferStream();
```

Comment:
Instead of triggering an update in refreshsharedui, how about we add an event in the loggers, something like "logavailable", and subscribe to that? There may be logs that don't correspond with a UI update, so we wouldn't see them until this method was called.